### DISCUSSION TITLE
tsdproxy.yaml

### DISCUSSION THREAD

**Discussion by Tom7320** [almeidapaulopt/tsdproxy#101] [Q&A] [answered]
Hi!

I installed TSDProxy according to the documentation, and it does work as expected. But there are a couple of things I do not understand - probably because of my only rudimentary knowledge about docker and proxies:

`targethostname` in docker section of tsdproxy.yaml:
Why do I need this? What IP address do I have to put there? The docker host's external IP address? Or the internal docker IP address of a container?

`controlurl` in tailscale section:
Why do I need this? What do I have to put there? Why/when do I have to change this?

`hostname`, `port` in http section:
What's this section for? Hostname and port of what?

Maybe someone has time and would like to answer my beginner questions. I would be very happy!

Have a nice evening

**Comment by almeidapaulopt** [comment, marked as answer]
hostname and port are internal to dashboard. you should onle need it in standalone usage.
controlurl is for headscape, if using tailscale, down touch it
targethostname is th IP address of your server. it's only used in rare cases, right now tsdproxy has a autedetect  ip and you should not need it

**Comment by Tom7320** [comment]
Thx a lot! Maybe it would make sense to add this information to the documentation to avoid confusion?
